lemur/certificates/views.py

```diff
@@ -678,17 +677,21 @@ def delete(self, certificate_id, data=None):
 
            .. sourcecode:: http
 
-              HTTP/1.1 200 OK
+              HTTP/1.1 204 OK
 
            :reqheader Authorization: OAuth token to authenticate
            :statuscode 204: no error
            :statuscode 403: unauthenticated
-           :statusoode 404: certificate not found
+           :statuscode 404: certificate not found
+           :statuscode 405: certificate deletion is disabled
 
         """
+        if not current_app.config.get('ALLOW_CERT_DELETION', False):
+            return dict(message="Certificate deletion is disabled"), 405
+
         cert = service.get(certificate_id)
 
-        if not cert:
+        if not cert or cert.deleted:
```

Comment:
Can you make a separate conditional? Something like:

`if cert.deleted:
    return dict(message="Certificate is already deleted"), 412`

Reply:
Done!